test/fixtures/accessible/main.js

```diff
@@ -0,0 +1,16 @@
+var app = require('electron').app
+var BrowserWindow = require('electron').BrowserWindow
+
+var mainWindow = null
+
+app.on('ready', function () {
+  mainWindow = new BrowserWindow({
+    center: true,
+    width: 800,
+    height: 400,
+    minHeight: 100,
```

Comment:
I copied another fixtures folder that had these set, they aren't explicitly needed by any of the new accessibility specs.